documentation/collectionruleexamples.md

```diff
@@ -4,39 +4,56 @@ The following examples provide sample scenarios for using a collection rule. The
 
 ## Collect Trace - Startup (`Startup` Trigger)
 
-### JSON
-
-```json
-{
-  "AssemblyLoadTraceOnStartup": {
-    "Trigger": {
-      "Type": "Startup"
-    },
-    "Actions": [
-      {
-        "Type": "CollectTrace",
-        "Settings": {
-          "Providers": [{
-              "Name": "Microsoft-Windows-DotNETRuntime",
-              "EventLevel": "Informational",
-              "Keywords": "0x8"
-          }],
-          "Duration": "00:00:15",
-          "Egress": "artifacts"
+<details>
+  <summary>JSON</summary>
+
+  ```json
```

Comment:
Some of these are indented to line up with the `<sumarry>` tag whereas other examples are no indented. Let's keep it consistent.